README.md

```diff
@@ -12,6 +12,8 @@ yarn dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
+Run `npm run prepare` for husky pre-commit hook.
```

Comment:
Hmm, looking at the README as a whole, we could do with rewriting it to reflect this project better. I won't let that block this, but I'll add a card at some point to update the whole README.